StripeFinancialConnections/StripeFinancialConnections/Source/Native/InstitutionPicker/InstitutionPickerViewController.swift

```diff
@@ -112,6 +112,7 @@ class InstitutionPickerViewController: UIViewController {
             target: self,
             action: #selector(didTapOutsideOfSearchBar)
         )
+        dismissSearchBarTapGestureRecognizer.cancelsTouchesInView = false
```

Comment:
this actually fixes any worries of blocking user interaction, but we still don't want the gesture recognizer to fire in some scenarios (ex. user taps "X" button in the search bar)